contrib/test_jenkins.sh

```diff
@@ -466,6 +466,13 @@ run_gtest() {
 	$AFFINITY $TIMEOUT make -C test/gtest test
 	(cd test/gtest && rename .tap _gtest.tap *.tap && mv *.tap $GTEST_REPORT_DIR)
 
+    echo "==== Running malloc hooks mallopt() test ===="
+    # the test is very short. run it on every shard.
+    env UCX_IB_RCACHE=n UCX_GTEST_MALLOPT=yes \
+        GTEST_SHARD_INDEX=0 GTEST_TOTAL_SHARDS=1 \
+        GTEST_FILTER=malloc_hook_cplusplus.mallopt make -C test/gtest test
+	(cd test/gtest && rename .tap _gtest.tap *.tap && mv *.tap $GTEST_REPORT_DIR)
```

Comment:
why is this needed?

Reply:
it seems to be a magic line that we do after ever gtest invocation

Reply:
i think this can override the previous tap files (of the main test - can you please check it)?

Reply:
I see that only malloc_hook_cplusplus.tap is created